### DISCUSSION TITLE
Issue on try Start Training

### DISCUSSION THREAD

**Discussion by j0fr3** [buliasz/tesstrain-windows-gui#40] [Q&A]
Hello @buliasz how are you? I'm trying to use your trainer, but I get an error on click start training:
![image](https://github.com/buliasz/tesstrain-windows-gui/assets/5282072/b20f5bf5-ab1f-49a4-bccd-c9bcab65a5c0)

---------------------------
Tesstrain GUI
---------------------------
Received 1 exit code for command: C:\WINDOWS\system32\cmd.exe /U /V:ON /C "set PYTHONIOENCODING=utf-8 && C:\Users\guest\AppData\Local\Programs\Python\Python312\python.exe C:\Users\guest\Desktop\tesstrain-windows-gui-main\generate_line_box.py -i "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.tif" -t "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.gt.txt" > "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.box""

'

If checkpoint files were created during the process you can use the 'Generate' button to generate '.traineddata' files.
---------------------------
OK   
---------------------------

Any idea how I can solve it? Thanks

**Comment by buliasz** [comment]
I guess you may have old version of tesseract. Can you check your version or try the latest one from [Tesseract at UB Mannheim](https://github.com/UB-Mannheim/tesseract/wiki) (I see version 5.3.3 at the moment available)?

If it doesn't help you can try copying the command shown to CMD and see what will be the result on your command line:
```
set PYTHONIOENCODING=utf-8 && C:\Users\guest\AppData\Local\Programs\Python\Python312\python.exe C:\Users\guest\Desktop\tesstrain-windows-gui-main\generate_line_box.py -i "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.tif" -t "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.gt.txt" > "C:\Users\guest\Desktop\tesstrain-windows-gui-main\ocr-pocket\12578445.box"
```
